angularjs-1.5-basics/src/app/interval-example/interval-example.controller.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->